spec/controllers/sites_controller_spec.rb

```diff
@@ -66,17 +66,22 @@ def make_request
         login_as site_manager
       end
 
-      it "displays the form" do
+      it "displays the form when requesting using an abbreviation" do
         get :edit, params: { id: site.abbr }
         expect(response.status).to eql(200)
       end
+
+      it "displays the form when requesting using an ID" do
+        get :edit, params: { id: site.id }
+        expect(response.status).to eql(200)
+      end
```

Comment:
Since we're offering backwards compatibility, shouldn't we also add an example like this to the specs for all the other controllers we're changing?

Reply:
Yes, will add a test to each as it looks like I forgot to do that.